3_run_rsync/templates/deploy.yml.j2

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ pod_name }}
+  namespace: {{ pvc_namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ pod_name }}
+  template:
+    metadata:
+      name: {{ pod_name }}
+      namespace: {{ pvc_namespace }}
+      labels:
+        app: {{ pod_name }}
+    spec:
+      imagePullPolicy: Always
```

Comment:
Do we want to add always pull? This has been painful for some people in the past.

Reply:
I like Always for this

Reply:
+1 for Always